packages/cli/src/commonlib/azureLogin.ts

```diff
@@ -647,6 +564,7 @@ async function listAll<T>(
 
 import AzureAccountProviderUserPassword from "./azureLoginUserPassword";
 import AzureLoginCI from "./azureLoginCI";
+import { AzureScopes } from "@microsoft/teamsfx-core";
```

Comment:
Move `import` to the top of the file.
Use the exact path "@microsoft/teamsfx-core/build/common/tools" to avoid dependency cycle. (Align with #6102 )